src/entities/settings/SettingsChangeResponse.java

```diff
@@ -0,0 +1,19 @@
+package entities.settings;
+
+public class SettingsChangeResponse {
+  private final String error;
+  private final Boolean isError;
+
+  public SettingsChangeResponse(String error, Boolean isError) {
```

Comment:
This doesn't make any sense. Pass a string `error` only, and have `isError` return `error != null`.

Reply:
Deleted this class